Zend/zend_exceptions.stub.php

```diff
@@ -24,7 +24,7 @@ class Exception implements Throwable
 {
     final private function __clone() {}
 
-    public function __construct(string $message = UNKNOWN, int $code = 0, ?Throwable $previous = null) {}
+    public function __construct(string $message = "", int $code = 0, ?Throwable $previous = null) {}
```

Comment:
It seems to me that we can declare the `""` default value with no problem. Or am I missing something?

Reply:
Extending Exception classes, will it change something?

Reply:
I was already thinking about it, but didn't find anything. However, now I found an issue:
```
class MyException extends Exception
{
    public function __construct() {
        $this->message = "abc";
        parent::__construct();
    }
}
```

In this case, `Exception::$message` will stay `"abc"`, rather than `""`. So we have to either make this parameter nullable, or change the implementation to always assign a value to `Exception::$message`

Reply:
This seems exotic enough that I'd just ignore the problem :)